### PR TITLE
Blockstack Todo Version 6.0

### PR DESCRIPTION
**Pull request by thinkverse**
@moxiegirl asked on the forum if someone could update this to use `blockstack@19` so I thought I'd submit a PR for it. I bumped up the major version because the internal dev dependencies now uses `@vue/cli` instead, fewer dependencies that way, updated `bootstrap@3` to `bootstrap@4`, also added a contributors list to `package.json` with as much info as I could find, feel free to update it yourselves.

Feel free to test this as much as you all can, especially the docker build, I don't have docker so cannot test that, same with firebase.

Tested the build, now moved to `/dist` instead of `/build`, using both `http-server` locally and on Netlify [here](https://silly-hoover-6aca37.netlify.com/)

### CODE REVIEW

**Comment by zone117x** [comment]
Awesome, thanks for doing this. and great work on dependency simplification. 👍 
I'll try to get this tested and reviewed today or tomorrow.

**Comment by moxiegirl** [comment]
This is great @thinkverse Thank you! I'm going to check this out today.

**Comment by thinkverse** [comment]
Yes, I've noticed that before, it seems like the config doesn't matter sometimes.

**Comment by moxiegirl** [comment]
@thinkverse Otherwise, LGTM on my part. I'll add an approving review once @zone117x or @hstove weighs in on the odd perms thing.

Thanks for cleanup on the dependencies.  Looked briefly again at that `tar` issue.  Seems like that is going to be fixable soon but nothing clean available now.  Which is to say, thanks for cleaning up all  the other cruft --- very much cleaner.

**Comment by moxiegirl** [comment]
Tested the compose file --- seems to be just there as a test environment.  Don't think we have any good documentation on how to use this Docker test environment with a sample. I'll dig into that but it shouldn't slow up merging this.

File: src/components/Landing.vue [hunk 1]
```diff
@@ -19,8 +19,8 @@ export default {
 
   methods: {
     signIn () {
-      const blockstack = this.blockstack
-      blockstack.redirectToSignIn()
+      const UserSession = new this.blockstack.UserSession(['store_write', 'publish_data'])
```
Comment (zone117x) [inline src/components/Landing.vue]: This should be changed to
```js
const appConfig = new this.blockstack.AppConfig(['store_write', 'publish_data'])
const UserSession = new this.blockstack.UserSession({ appConfig: appConfig })
```

Comment (thinkverse) [reply src/components/Landing.vue]: Does that need to be changed for every instance or do you want the change just here? This also seems to fix the scope issue we where having of the right permissions not showing up, might be a good idea to emphasize the importance of the `appConfig` in the docs.

Comment (zone117x) [reply src/components/Landing.vue]: Only needs changed here since the rest of the UserSession usage is using defaults.

Comment (thinkverse) [reply src/components/Landing.vue]: Okay, pushing a fix now.

**Comment by moxiegirl** [comment]
@thinkverse Thanks for the work here. I think two reviewers are enough and I know @hstove is super busy --- and @zone117x is very detail oriented. Thanks for the work.

@zone117x I'll clarify the permission scopes in the doc I'm working on (out for the review later tonight)

**Pull request merged**